src/pages/add-lock-exception/AddLockExceptionForm/OrganisationUnitSelection/use-org-unit-cache.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
+
+const useOrgUnitCache = () => {
```

Comment:
This hook reimplements the [caches used by the `d2-ui` version of the OrgUnitSelect components](https://github.com/dhis2/d2-ui/blob/master/packages/org-unit-select/src/OrgUnitSelectByLevel.component.js#L15).